01_fib/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	//fmt.Fprintln(out, " Fibnacci Series")
+	fib(7)
+}
+
+func fib(limit int) {
+	firstNo := 0
+	secondNo := firstNo + 1
```

Comment:
You can fold this into one line: `firstNo, secondNo := 0, 1`.

Reply:
Folded in Single Line